core/commands/generate_command.rb

```diff
@@ -447,6 +447,14 @@ def generate_vagrant_file(path, config, boxes, provider, cookbook_path)
       end
     end
     vagrant.puts vagrant_config_footer
+  rescue RuntimeError => e
+    @ui.error(e.message)
+    @ui.error('Configuration is invalid')
+    FileUtils.rm_rf(path)
```

Comment:
You remove the directory while the vagrant file is still open.

Reply:
Also what happens to the AWS credentials that were generated for this configuration?